plugins/to.py

```diff
@@ -15,7 +15,15 @@ def setup(irc):
                 ("unixtime", int),
                 ("channel", str),
                 ("message", str))
-        # TODO: Drop messages more than a year old
+        else:
+            # Drop messages more than a year old
+            now = int(time.time())
+            one_year = 60 * 60 * 24 * 366
+            query = "SELECT * FROM saxo_to"
+            for row in db.query(query):
```

Comment:
Wouldn't it be better to add a `WHERE` clause here instead of selecting all messages? Or even better just `DELETE .. WHERE`